src/main/java/pro/belbix/ethparser/properties/AppProperties.java

```diff
@@ -35,10 +35,12 @@
     private boolean parsePrices = true;
 
     private String startBlock = "";
-    private boolean parseTransactions = false;
     private boolean parseHarvest = false;
+    private boolean parseUniswapTransactions = false;
+    private boolean parseDeployerTransactions = false;
 
     private boolean parseLog = true;
+    private boolean parseTransactions = false;
```

Comment:
now we will have an active transaction parser so you can switch on by default transaction parsing and deployer parser

Reply:
I had this false because that is what it was before I organized it in class ... did I need any changes here?

Reply:
I will do myself when prepeare release, don't worry